install-and-configure/advanced-configuration/controller/kubescaler.md

```diff
@@ -33,15 +33,9 @@ Notable Helm values:
 
 ### Supported workload types
 
-Kubescaler supports:
+Kubescaler supports apps/v1 Deployments.
 
-* apps/v1 Deployments
-* apps/v1 DaemonSets
-* batch/v1 CronJobs (K8s v1.21+). No attempt will be made to autoscale a CronJob until it has run at least once.
```

Comment:
Was there a recent discussion that sparked this change?

Reply:
Not really, I just noticed that the docs on this page were inconsistent. It says at the top that only deployments were supported, and in this section it contradicts that statement. Any problems you see with this?

Reply:
Nope! After further research it looks like this change is correct according to our internal docs on it https://github.com/kubecost/cluster-controller/blob/develop/docs/kubescaler.md

Reply:
Thanks for the checkup.